srcpkgs/ibus-bamboo/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'ibus-bamboo'
+pkgname=ibus-bamboo
+version=0.4.6
+revision=1
+build_style=gnu-makefile
+short_desc="Vietnamese input method for ibus."
+maintainer="luongthanhlam"
+license="GPL-3"
+homepage="https://github.com/BambooEngine/ibus-bamboo"
+distfiles="https://github.com/BambooEngine/ibus-bamboo/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
+checksum="2345394fd1fd8de5a9f620f3775ab34c3233c16eb6a9ffe166e3ed6f1a31f1bb"
+hostmakedepends="make go libXtst-devel libX11-devel"
+archs=noarch
+depends="ibus"
```

Comment:
this should go directly below makedepends= that should go directly below hostmakedepends= that should go directly below build_style=